pkg/agent/openflow/pipeline.go

```diff
@@ -448,11 +558,15 @@ func (c *client) l3ToGWFlow(gwMAC net.HardwareAddr, category cookie.Category) bi
 // l3ToGatewayFlow generates flow that rewrites MAC of the packet received from tunnel port and destined to local gateway.
 func (c *client) l3ToGatewayFlow(localGatewayIP net.IP, localGatewayMAC net.HardwareAddr, category cookie.Category) binding.Flow {
 	l3FwdTable := c.pipeline[l3ForwardingTable]
-	return l3FwdTable.BuildFlow(priorityNormal).MatchProtocol(binding.ProtocolIP).
+	nextTable := l3FwdTable.GetNext()
+	if c.enableProxy {
+		nextTable = l2ForwardingCalcTable
```

Comment:
Why the next table is localPodL3ForwardingTable when enableProxy=false?

Reply:
The table is also splitted in origin mode.

Reply:
Got your point, thanks.
Updated.